store.go

```diff
@@ -14,10 +14,10 @@ type updateFn[V any] func(cur, prev V) bool
 
 // TODO: Do we need this to be a separate struct from Item?
 type storeItem[V any] struct {
+	expiration time.Time
+	value      V
```

Comment:
Here, value V should be at the end because it is variable length.

Reply:
I dropped the alignment changes from this PR.
I need to see what affect generics has on struct alignment before proposing them again.